packages/tupaia-web/src/features/Chart.tsx

```diff
@@ -3,51 +3,128 @@
  * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
  */
 
-import React from 'react';
+import React, { ChangeEvent, useState } from 'react';
 import styled from 'styled-components';
-import { Chart as ChartComponent, ViewContent } from '@tupaia/ui-chart-components';
+import { Chart as ChartComponent, ChartTable, ViewContent } from '@tupaia/ui-chart-components';
+import { BarChart, GridOn } from '@material-ui/icons';
+import { Tabs } from '@material-ui/core';
+import { Tab } from '@material-ui/core';
+import { TabContext, TabPanel } from '@material-ui/lab';
 
-const Wrapper = styled.div<{
-  $isEnlarged: boolean;
-  $hasData: boolean;
-}>`
+const Wrapper = styled.div`
   display: flex;
   position: relative;
   align-content: stretch;
   -webkit-box-align: stretch;
   align-items: stretch;
-  height: ${({ $isEnlarged, $hasData }) => {
```

Comment:
this line is fine- just noticed that webkit selector above which is now redundant.